tools/download_from_gcs.py

```diff
@@ -130,7 +126,7 @@ def MaybeDownloadDirectoryFromGcs(bucket,
   return res
 
 
-if __name__ == '__main__':
+def main():
   logging_format = '[%(levelname)s:%(filename)s:%(lineno)s] %(message)s'
```

Comment:
I think we should only configure logging when we're running this as a standalone script instead of importing it, but the rest makes sense here

Reply:
Are you suggesting to keep these two lines down in `if __name__ == '__main__':`?

If someone imports this script I suppose they could run main() but that seems awfully wrong to do.

Reply:
Yeah, that's what I mean—I don't expect anyone to import it, but I still think it's a good pattern to keep it out. If you disagree for some reason then you can keep it, this is just my preference

Reply:
Done